openwisp-utils-qa-checks

```diff
@@ -83,15 +83,18 @@ runisort() {
     || { echoerr "ERROR: Isort check failed!"; FAILURE=1; }
 }
 
+# This check is run when any of the following conditions is true:
+# 1. --force-checkcommit is specified
+# 2. This function is invoked from Travis building a Pull Request
 runcheckcommit() {
-  [ -n "$TRAVIS_PULL_REQUEST" ] || TRAVIS_PULL_REQUEST="false"
-  if [ "$SKIP_CHECKCOMMIT" = "false" -o "$TRAVIS_PULL_REQUEST" != "false" ]; then
+	if [[ $FORCE_CHECKCOMMIT = true ]] ||
```

Comment:
indentation is odd, can you put this at the same indentation level as it was before please?